modules/ROOT/pages/mule-credentials-vault.adoc

```diff
@@ -344,7 +344,11 @@ Administrative commands such as the `ps` command in UN*X or the `tasklist` comma
 
 Though the scenarios just mentioned satisfy most of the use cases, there are situations where you need to hide the secret encryption key value from other users of the operating system, that is, you must not show the secret encryption key as a JVM parameter, since it is visible to anyone having access to the process.
 
-A solution for this is to implement a custom secure property placeholder that can read the encryption key in a different way, for example from a file located in a protected folder. This custom implementation must extend `org.mule.modules.security.placeholder.SecurePropertyPlaceholderModule`.
+One solution for this is to implement a custom secure property placeholder that can read the encryption key in a different way, for example from a file located in a protected folder. This custom implementation must extend `org.mule.modules.security.placeholder.SecurePropertyPlaceholderModule`.
+
+Another solution is to set the `mule.runtime.verbose` flag to `false` at Mule startup in order to a void that encryption keys are logged with the system properties.
```

Comment:
```suggestion
Another solution is to set the `mule.runtime.verbose` flag to `false` at Mule startup, so that encryption keys are not logged with the system properties.
```
Please verify that this rewording is still technically accurate.